site/src/pages/docs/advanced-streaming-backpressure.mdx

```diff
@@ -31,14 +26,13 @@ Server.builder()
       .service("/big_file.dat", bigFile.asService())
       .build();
 ```
-The <type://HttpFile> loads chunks one by one and send them to the client with backpressure.
 
-## Sending a streaming response using <type://HttpResponseWriter>
```

Comment:
So it's not a good idea to use a link in the subtitle?

Reply:
Some sites have clickable headings which points to themselves so users can copy a link straight away from the URL field of a browser. Considering that many tools these days add links to headings  automatically (like  Armeria site providing a permalink) by SSG, adding links to a heading doesn't seems like a good practice. Although Armeria overcomes this and generates an anchor that works perfectly well, having the term "type" attached to the HttpResponseWriter may be confusing in terms of accsesibility.

Reply:
That totally makes sense. We have a link for the type in the section anyway. 😄